### ISSUE TITLE
Tiny Petition: Seagull don't eat certain items config

### ISSUE THREAD

**Issue by Morthalix** [Alex-the-666/AlexsMobs#2133]
I am scared of seagull. I use a mod that adds a lunch bag, where i put 3, 6, 9 stacks of food.
It's a item that when the seagull steal it, i lost all the food, not a single, because everything is in the bag. I ask for a config that is a blacklist for the seagull, that he don't steal it.
Thank you very much...i love the mod, that's the only pebble in the foot.

**Comment by Morthalix** [comment]
Writing this, dont let die the petition

**Comment by ChicknTurtle** [comment]
I agree, seagulls shouldn't be able to eat artifact's eternal steak and everlasting beef. There is a config to blacklist items from being stolen from players, but nothing to change what can be picked up from the ground and eaten.

**Comment by XLRC888** [comment]
i agree with this.
